c/kastore.h

```diff
@@ -135,12 +139,12 @@ The library major version. Incremented when breaking changes to the API or ABI a
 introduced. This includes any changes to the signatures of functions and the
 sizes and types of externally visible structs.
 */
-#define KAS_VERSION_MAJOR   1
+#define KAS_VERSION_MAJOR   2
```

Comment:
Do we need a major version bump here? We're just changing the semantics of errors slightly I would have thought, which we can probably make do with a minor version bump for (seems unlikely people are depending on the exact error code returned).